util/functions.js

```diff
@@ -135,7 +135,7 @@ module.exports = (client) => {
   // enmap should only have *unique* overrides that are different from defaults.
   client.getSettings = (guild) => {
     if(!guild) return client.settings.get("default");
-    const guildConf = client.settings.get(guild.id) || {};
+    const guildConf = client.settings.ensure(guild.id, client.settings.get("default"));
     // This "..." thing is the "Spread Operator". It's awesome!
     // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax
     return ({...client.settings.get("default"), ...guildConf});
```

Comment:
I assume the ensure method clones the defauult settings (I don't know, never used Enmap), but if so, this could be omitted alongside with the comment above.

```suggestion
     return guildConf;
```

In fact, this could be refactored completely to:
```javascript
client.getSettings = guild => guild ?
  client.settings.ensure(guild.id, client.settings.get("default")) :
  client.settings.get("default");
```

Reply:
cc @eslachance

Reply:
It does not. It _ensures_ that the server settings are there.